src/Umbraco.Core/PropertyEditors/IDataEditorWithMediaPath.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Umbraco.Core.PropertyEditors
+{
+    public interface IDataEditorWithMediaPath : IDataEditor
```

Comment:
It's probably easier/nicer and more flexible that IDataEditorWithMediaPath doesn't inherit from IDataEditor and it's just a standalone interface.

Reply:
I've removed the inheritance